README.md

```diff
@@ -49,29 +58,61 @@ modules.
 - Test::More
 - Test::Needs
 - Test::Notabs
+- Test::Perl::Critic
 - Test::Pod
 - Test::Pod::Coverage
 - Test::Portability::Files
 - Test::RequiresInternet
 - Test::Simple
 - Test::Spelling
 - Test::Synopsis
+- Test::Vars
 - Test::Version
 - Test::Warnings
 
-## Only on Perl 5.10 and later
+## Only on Perl 5.10
```

Comment:
Maybe we could clarify about these. We are pinning to specific versions on 5.10 as those should be the last 5.10 supported versions of those modules. They should also get installed in more recent Perls, but we haven't specified a version requirement. If they're not specifically listed elsewhere, I assume they are pulled in via dependencies or they were at the time when the change was made.

Reply:
Yes, I thought the same. I'll verify that these are actually pulled in as dependencies and explicitly list them in the _cpanfile_. This means that fewer cases must be distinguished in _Readme.mk_.